packages/runtime/runtime-utils/src/summarizerNode.ts

```diff
@@ -778,3 +548,32 @@ export class SummarizerNode implements ISummarizerNode {
         }
     }
 }
+
+/**
+ * Creates a root summarizer node.
+ * @param logger - Logger to use within SummarizerNode
+ * @param summarizeInternalFn - Function to generate summary
+ * @param changeSequenceNumber - Sequence number of latest change to new node/subtree
+ * @param referenceSequenceNumber - Reference sequence number of last acked summary,
+ * or undefined if not loaded from summary
+ * @param config - Configure behavior of summarizer node
+ */
+export const createRootSummarizerNode = (
+    logger: ITelemetryLogger,
+    /** Summarize function */
+    summarizeInternalFn: (fullTree: boolean) => Promise<ISummarizeInternalResult>,
+    /** Sequence number of latest change to new node/subtree */
+    changeSequenceNumber: number,
+    /**
```

Comment:
You can remove comments from here since they are in the comment block before the function.

Reply:
Good catch, removed those.